src/SprykerEco/Service/ComputopApi/ComputopApiServiceInterface.php

```diff
@@ -145,6 +147,7 @@ public function generateReqIdFromQuoteTransfer(QuoteTransfer $quoteTransfer);
 
     /**
      * Specification:
+     * - Requires `QuoteTransfer.totals` and `QuoteTransfer.customer`
```

Comment:
same as above 
and dot at the end

Reply:
Fixed

Reply:
please ensure every line of API spec ends with a dot
check touched lines
I see several ones without dot at he end

Reply:
Fixed